tests/src/test/java/org/corfudb/test/AbstractCorfuUniverseTest.java

```diff
@@ -111,9 +111,9 @@ public DockerCorfuCluster getDefaultCrossVersionCluster(DockerUniverseWorkflow w
         String networkName = wf.getUniverse().getUniverseParams().getNetworkName();
 
         DockerContainerParams<CorfuServerParams> containerParamsV1 = getContainerParams(
-                corfuCluster, networkName, 9005, "latest");
+                corfuCluster, networkName, 9005, "0.3.1-SNAPSHOT");
```

Comment:
change back?

Reply:
yeah will be changed back, just for local testing